src/test/unit/ntypes/SdrTest.cpp

```diff
@@ -718,3 +718,9 @@ TEST(SdrTest, TestSaveLoad) {
     ASSERT_TRUE( flat    == flat_2 );
     ASSERT_TRUE( index   == index_2 );
 }
+
+TEST(SdrTest, TestProxy) {
+    SDR A({10});
+    SDR_Proxy B(A, {2, 5});
+    SDR *C = &B;
```

Comment:
:+1: 
add test that getters work
add test that setters throw : EXPECT_THROW(..)
show how you can access the class with different dimensions

Reply:
this also errs CI as unused variable C